deps/wazuh_testing/wazuh_testing/tools/performance/visualization.py

```diff
@@ -48,6 +48,9 @@ def __init__(self, dataframes, target, compare=False, store_path=gettempdir(), x
         if target in ['binary', 'analysis', 'remote', 'agent', 'logcollector', 'wazuhdb']:
             self.columns_to_plot = self._load_columns_to_plot(columns_path)
 
+        if unify.lower() in ["true"]:
+            self._unify_dataframes()
```

Comment:
```suggestion
        if unify_child_daemons_metrics:
            self._unify_dataframes()
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5500/commits/4f128638fdbf2976fae270c15f42731b033bddf2